### DISCUSSION TITLE
Trakt Watchlist add - watched episode status

### DISCUSSION THREAD

**Discussion by jeffski10** [pymedusa/Medusa#8085] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
I have found a number of instances where a show is removed from Medusa as it has finished but then a new series appears and gets listed.  Whilst I had to add this back in Trakt to the watchlist, Medusa then will download all previously watched episodes. 

**Describe the solution you'd like**
It would be great to be able to set a status for previous watched v unwatched so these could be marked as skipped/archived. Saves downloading them again and then ignoring/deleting.  Maybe as part of the episode sync process.

**Comment by p0psicles** [comment]
I'm not sure what your issue is. You download series a, remove it in medusa. Then added it through trakt. And have trakt syncing enabled? Then it's added back to medusa?

**Comment by p0psicles** [comment]
Can you explain your your work flow in detail?

**Comment by jeffski10** [comment]
So, take an example - the Good Fight which restarted recently.  I had this on my watchlist, was added to medusa for the previous seasons. I then watch said episodes and have a cleanup run on Kodi to remove watched episodes. This then flips to archived in medusa so all is good. 
When the season ended and I had watched there were no future episodes listed on the TVDB so it appeared to be 100% watched and gets removed from medusa and trakt removes from the watchlist automatically.   6-9 months later new episodes are announced and get added to the DB. I spot this recently when the season restarts and add the show back to the trakt watchlist. At this point it will get picked up and sync again but pull down previously watched episodes. 
(I accept this is the fault of the data sets being used -likely where show status is not set to Continuing at some point). So little I can do about it dropping off the watchlist but it would be nice to recover without as large a download albeit it just goes through the loop above and gets deleted again. 
I also suffered from this when I did a clean rebuild of the install after there was some DB corruption.

**Comment by p0psicles** [comment]
If you delete the shows episodes and later the show itself. There is no way to track the show?

**Comment by jeffski10** [comment]
But when the Traktchecker looks for /syncs the wanted episodes can it read the watched flag on the episode so as not to see it as wanted any more?

**Comment by p0psicles** [comment]
Aah ok I understand. That would be a new option. Which should be opt-in. Really low priority though.
